### PR TITLE
Linked to problem

### PR DESCRIPTION
**Pull request by raksonibs**
A small issue with {{link-to}} I noticed. Just to keep things consistent! Working on a Spanish translation now by the way!

### CODE REVIEW

File: source/localizable/tutorial/routes-and-templates.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ When we look at our about page at `http://localhost:4200/about`, we now have a w
 
 ![super rentals about page screenshot](../../images/routes-and-templates/ember-super-rentals-about.png)
 
-Now, we'll add a link to our contact page so we can navigate from back and forth between `about` and `contact`.
```
Comment (locks) [inline source/localizable/tutorial/routes-and-templates.md]: This change isn't correct, we really do mean "add a link to", we just happen to use the `link-to` helper.

File: source/localizable/applications/services.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ To access a service,
 you can inject it in any container-resolved object such as a component or another service using the `Ember.inject.service` function.
 There are 2 ways to use this function.
 You can either invoke it with no arguments, or you can pass it the registered name of the service.
-When no arguments are passed the services is loaded based in the name of the variable key.
```
Comment (locks) [inline source/localizable/applications/services.md]: Please change the text only in this version and leave the others untouched, CrowdIn will synchronize properly :)

**Comment by acorncom** [comment]
@raksonibs Looks like we've got a few merge conflicts here.  Would you mind rebasing and removing your changes to the translated files?

**Comment by locks** [comment]
Fixed through 2ae5bf28ba614d01c62d80c0fb068fdc4228c8ea.

**Pull request closed without merging**